views/list_project.erb

```diff
@@ -16,13 +16,15 @@
       <th>Hipchat Channel</th>
       <th>Mail Address</th>
       <th>Webhook URL</th>
+      <th>Access Token</th>
     </tr>
     <% @projects.each do |project| %>
       <tr>
         <td><a href="<%= to("/project/#{ project.id }") %>"><%= project.name %></a></td>
         <td><%= project.hipchat_channel %></td>
         <td><%= project.mail_address %></td>
         <td><%= project.webhook_url %></td>
+        <td>***</td>
```

Comment:
Same comment for the other places this shows up. Should be an easy fix though :)